README.md

```diff
@@ -251,6 +251,24 @@ class SignupForm < ApplicationForm
 end
 ```
 
+### Upload fields
+If you want to add file upload fields to your form you will need to initialize your form with the `enctype` attribute set to `multipart/form-data` as shown in the following example code:
+
+```ruby
+class User::ImageForm < ApplicationForm
+  def template
+    # render label
+    render field(:image).upload.label { "Choose file" }
```

Comment:
Could the `.upload` methods be changed to `input(type: "file")` to keep the example "close to HTML" or is there a reason a dedicated component is needed?

Reply:
Hey Brad,
ok changed the example. There was no particular reason to use a component. I agree that using input is better for the docs. Updated the PR…